frontend/interfaces/software.ts

```diff
@@ -42,7 +42,7 @@ export interface ISoftware {
   hosts_count?: number;
   last_opened_at?: string | null; // e.g., "2021-08-18T15:11:35Z”
   installed_paths?: string[];
-  browser?: string;
+  extension_for?: string;
```

Comment:
Since we're still including the `browser` field in the places it was previously returned seems like
it would be good to keep that field here as well, with a note saying it still exists for backwards
compatibility only maybe?

Reply:
Actually I guess this is the situation where `browser` was never present, so okay to replace here, but I think we want
to keep `browser` in the places it's still being returned. Looks like `ISoftwareTitle`, `ISoftwareTitleDetails`, `ISoftwareVersion`